tests/dracula/test_run_topp.py

```diff
@@ -102,4 +84,113 @@
+#     plt.xlabel("Path position")
+#     plt.ylabel("Path velocity square")
+#     plt.legend()
+#     plt.tight_layout()
+#     plt.show()
+
+#    if debugging:
+#         plt.figure()
+#         s_sampled = np.linspace(0, csplcp.x[-1], 100)
+#         fig, axs = plt.subplots(1, 4, sharex=True, figsize=[18, 4])
+#         for i in range(csplcp.c.shape[2]):
+#             axs[0].plot(
+#                 s_sampled, csplcp(s_sampled)[:, i], label="J{:d}".format(i + 1)
+#             )
+#             axs[1].plot(
+#                 s_sampled,
+#                 csplcp(s_sampled, 1)[:, i],
+#                 label="J{:d}".format(i + 1),
+#             )
+#             axs[2].plot(
+#                 s_sampled,
+#                 csplcp(s_sampled, 2)[:, i],
+#                 label="J{:d}".format(i + 1),
+#             )
+#             axs[3].plot(
+#                 s_sampled,
+#                 csplcp(s_sampled, 3)[:, i],
+#                 label="J{:d}".format(i + 1),
+#             )
+#         axs[0].set_xlabel("Time (s)")
+#         axs[0].set_ylabel("Joint position (rad)")
+#         axs[1].set_xlabel("Time (s)")
+#         axs[1].set_ylabel("Joint velocity (rad/s)")
+#         axs[2].set_xlabel("Time (s)")
+#         axs[2].set_ylabel("Joint acceleration (rad/s^2)")
+#         axs[3].set_xlabel("Time (s)")
+#         axs[3].set_ylabel("Joint jerk (rad/s^3)")
+#         axs[0].legend()
+#         axs[1].legend()
+#         axs[2].legend()
+#         axs[3].legend()
+#         plt.tight_layout()
+#         fig.suptitle("original")
+#         plt.show()
+#         s_sampled2 = np.linspace(0, cspl.x[-1], 100)
+#         fig, axs = plt.subplots(1, 4, sharex=True, figsize=[18, 4])
+#         for i in range(cspl.c.shape[2]):
+#             axs[0].plot(
+#                 s_sampled2, cspl(s_sampled2)[:, i], label="J{:d}".format(i + 1)
+#             )
+#             axs[1].plot(
+#                 s_sampled2,
+#                 cspl(s_sampled2, 1)[:, i],
+#                 label="J{:d}".format(i + 1),
+#             )
+#             axs[2].plot(
+#                 s_sampled2,
+#                 cspl(s_sampled2, 2)[:, i],
+#                 label="J{:d}".format(i + 1),
+#             )
+#             axs[3].plot(
+#                 s_sampled2,
+#                 cspl(s_sampled2, 3)[:, i],
+#                 label="J{:d}".format(i + 1),
+#             )
+#         axs[0].set_xlabel("Time (s)")
+#         axs[0].set_ylabel("Joint position (rad)")
+#         axs[1].set_xlabel("Time (s)")
+#         axs[1].set_ylabel("Joint velocity (rad/s)")
+#         axs[2].set_xlabel("Time (s)")
+#         axs[2].set_ylabel("Joint acceleration (rad/s^2)")
+#         axs[3].set_xlabel("Time (s)")
+#         axs[3].set_ylabel("Joint jerk (rad/s^3)")
+#         axs[0].legend()
+#         axs[1].legend()
+#         axs[2].legend()
+#         axs[3].legend()
+#         plt.tight_layout()
+#         fig.suptitle("new")
+#         plt.show()
```

Comment:
Delete or uncomment commented code

Reply:
it may be useful for writing tests and debugging. but i don't have time to finish that right now so copied over and keeping it